app/views/events/show.html.erb

```diff
@@ -50,16 +50,16 @@
               <%= f.submit 'Afwezig', class: 'cursor-pointer ml-4 inline-flex items-center px-4 py-2 border border-transparent text-sm font-bold rounded-md shadow-sm text-white bg-gray-400 hover:bg-gray-500 focus:outline-none focus:ring-2 focus:ring-gray-500' %>
             </div>
           <% end %>
-          <p class="text-center sm:text-right text-gray-500 py-2">Inschrijven kan tot <%= @event.deadline&.strftime('%d-%m-%Y %H:%M') || @event.date.strftime('%d-%m-%Y %H:%M') %></p>
+          <p class="text-center sm:text-right text-gray-500 py-2">Inschrijven kan tot <%= (@event.deadline || @event.date)&.strftime('%d-%m-%Y %H:%M') %></p>
```

Comment:
Hij zou altijd een date moeten hebben daar doen we te veel mee om nil toe te staan. Dan is de safety operator niet meer nodig.